dashboard/hack/start_nginx.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+inject_vite_envs.sh
+
```

Comment:
```suggestion
envsubst '${ENDURO_API_ADDRESS}' <  /etc/nginx/templates/default.conf.template > /etc/nginx/conf.d/default.conf
```

We need to inject the address for the enduro api